content/2017/06/27/susy3.rst

```diff
@@ -0,0 +1,871 @@
+    <script async src="https://production-assets.codepen.io/assets/embed/ei.js"></script>
+  </figure>
+
+That will also work with css-tables, etc.
+Add any padding you like,
+and you have on-the-fly fluid grids
+without any third-party tools.
+
+We've even built a nearly-complete Susy3
+in pure CSS.
+This is a fun experiment,
+but I don't recommend using it
+in production:
+
+.. raw:: html
+
+  <figure class="extend-small">
+    <p data-height="500" data-theme-id="0" data-slug-hash="PboVrw" data-default-tab="result" data-user="mirisuzanne" data-embed-version="2" data-pen-title="SusyCSS Demo" data-editable="true" class="codepen">See the Pen <a href="https://codepen.io/mirisuzanne/pen/PboVrw/">SusyCSS Demo</a> by Miriam Suzanne (<a href="https://codepen.io/mirisuzanne">@mirisuzanne</a>) on <a href="https://codepen.io">CodePen</a>.</p>
+    <script async src="https://production-assets.codepen.io/assets/embed/ei.js"></script>
+  </figure>
+
+Susy is designed to handle any layout,
+but if you are designing grids in pure CSS,
+it's better to describe your actual needs
+more simply.
+Why solve all layout problems,
+when you can solve only the problems you have?
+
+
+Use Flexbox
+~~~~~~~~~~~
+
+The great thing about flexbox
+is that it allows items to flex in specific relationship
+with each other,
+including equal-height columns and vertical centering.
+I often find that I don't need Susy for a flexbox layout,
+because I'd rather define relationships
+instead of explicit grid-columns.
+
+Here's an example flexbox layout,
+without Susy:
+
+.. raw:: html
+
+  <figure class="extend-small">
+    <p data-height="450" data-theme-id="0" data-slug-hash="657a71f05b9c044d0235bab212abdbdc" data-default-tab="css,result" data-user="mirisuzanne" data-embed-version="2" data-pen-title="Full-height Flexbox" class="codepen">See the Pen <a href="https://codepen.io/mirisuzanne/pen/657a71f05b9c044d0235bab212abdbdc/">Full-height Flexbox</a> by Miriam Suzanne (<a href="https://codepen.io/mirisuzanne">@mirisuzanne</a>) on <a href="https://codepen.io">CodePen</a>.</p>
+    <script async src="https://production-assets.codepen.io/assets/embed/ei.js"></script>
+  </figure>
+
+If you do want to use Susy with flexbox
+to achieve more consistent grids,
+you only need to replace any ``flex-basis`` values
+with Susy's ``span()`` function.
+
+.. code:: scss
+
+  .flex {
+    flex: 1 1 span(3 of 12);
+  }
+
+I can't think of any way to improve that with mixins,
+unless you want a few flexing defaults:
+
+.. code:: scss
+
+  @mixin span(
+    $span,
+    $config: ()
+  ) {
+    flex: 1 1 span($span, $config);
+    // split the gutter on each side of the element…
+    padding: gutter($span, $config) / 2;
+  }
+
+
+Use Calc to Mix Units
+~~~~~~~~~~~~~~~~~~~~~
+
+Calc can also help you with mixed-unit grids,
+combining fluid and fixed columns and gutters.
```

Comment:
wonder if we should suggest a good use case for fixed width columns, like hosting ads on your site since most ad networks have fixed sized ads still.